tests/test_blog_api.py

```diff
@@ -35,6 +35,7 @@ def test_get_articles_with_transforming_links(self):
         self.assertIn(
             "ubuntu.com/wp-content/uploads", article["content"]["rendered"]
         )
+        self.assertIn("cloudinary", article["content"]["rendered"])
```

Comment:
Could we test it actually does the transfromation we expect?

Reply:
> A question - does this only change images in the body of the article? What about the thumbnails on the index page?
> 
> I know that at the moment in e.g. `jp.ubuntu.com` the cloudinary prefix is added to thumbnails inside the template, but this isn't ideal. Really, if this module is going to do the transformation anywhere it should do it everywhere, and the prefix should be removed from templates.
> 
> That also therefore means that you need to make this at least another minor version change, as it effects the module's API.

This works now. Thumbnails are affected by the new `enable_cloudinary` parameter too.

Reply:
No idea why the reply was put under Joao's comment.